From abbdec143f593990f2d47b5146c948f58ca30f71 Mon Sep 17 00:00:00 2001
From: root <root@vpn01.ffv>
Date: Sun, 4 Dec 2016 17:25:54 +0100
Subject: [PATCH] Don't force links to be unidirectional between nodes

Hopglass wants its links to be bidirectional. This information
is used to display detailed information about the link quality
in both directions.
---
 backend.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backend.py b/backend.py
index 27335d0..62185d0 100755
--- a/backend.py
+++ b/backend.py
@@ -140,7 +140,7 @@ def main(params):
     graph.mark_vpn(batadv_graph, extract_tunnel(nodedb['nodes']))
 
     batadv_graph = graph.merge_nodes(batadv_graph)
-    batadv_graph = graph.to_undirected(batadv_graph)
+    #batadv_graph = graph.to_undirected(batadv_graph)
 
     # write processed data to temporary files in dest dir
     # and rename them atomically to prevent inconsistent data
-- 
2.1.4

